src/web_app_skeleton/docker/development.dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM crystallang/crystal:1.1.1
+
+# Add the nodesource ppa to apt.
+RUN wget https://deb.nodesource.com/setup_16.x -O- | bash
+
+# Install nodejs, npm, yarn
+RUN apt-get update
+RUN apt-get install -y nodejs
+RUN npm install -g yarn mix
+
+# Install postgres cli tools
+RUN apt-get install -y postgresql-client
```

Comment:
I think I agree, but I'm not sure. In a practical example of development, other packages are going to be added to this file over the lifetime of an app. Since there are multiple installs taking place, cleaning the apt-cache should happen below all of them.

I could add it to the bottom of the dockerfile, but I'm not sure it's worth it. At some point someone is going to need to shell into the container and check on something and if they need to install it'll be two steps rather than just one. Leaving it out means their debugging might be faster.

Since this is a development container, is the space savings worth it?